packages/mdc-toolbar/mdc-toolbar.scss

```diff
@@ -78,13 +77,26 @@ $mdc-toolbar-mobile-breakpoint: 599px;
   &__title {
     @include mdc-typography(title);
 
+    align-self: center;
     margin: 0;
+    margin-left: 16px;
+    padding: $mdc-toobar-element-vertical-padding 0;
```

Comment:
We could. _Technically_ that's not what we want though. I want the title element to be 16px away from the icon element. We _don't_ want the title element to have 16px of padding.